src/FFMpeg/Format/Video/DefaultVideo.php

```diff
@@ -32,8 +33,56 @@
     /** @var Integer */
     protected $modulus = 16;
 
-    /** @var Array */
-    protected $additionalParamaters;
+    /** @var string */
+    private $profile = Profile::MAIN;
+
+    /** @var float */
+    private $level = 3.1;
+
+    /** @var string[] */
```

Comment:
Overall, I'm fine with your additions.
I think this var declaration should be Array and not string[] though, but this can be fixed later.